shared/tinyusb/mp_usbd_cdc.c

```diff
@@ -68,7 +72,7 @@ uintptr_t mp_usbd_cdc_poll_interfaces(uintptr_t poll_flags) {
     return ret;
 }
 
-void tud_cdc_rx_cb(uint8_t itf) {
+void MICROPY_TINYUSB_CALLBACK(tud_cdc_rx_cb)(uint8_t itf) {
```

Comment:
Note that `tud_cdc_rx_cb` is called from `mp_usbd_cdc_poll_interfaces()`... not sure if you want the wrapped version to be called, or the raw one?

Reply:
Good catch! The wrapped version should be called. However, I also wrap:
```C
           -Wl,--wrap=mp_hal_stdio_poll \
           -Wl,--wrap=mp_hal_stdout_tx_strn
```
So I don't think it's a problem, but I will fix it anyway.